page-index/page-index.js

```diff
@@ -687,6 +687,33 @@ const pageIndex = [
     type: PAGE,
     href: "tutorials",
     children: [
+      {
+        Title: "Getting Started with Promscale",
```

Comment:
I don't really care, but wondering if others have any opinion on where this ends up in the tree. I put it at the top so that it was easy to locate locally, and I also remember @avthars mentioning maybe adding steps/numbers here.

Just wanted to make sure this was what we wanted.

Reply:
I think this should go below Hello Timescale

Reply:
Second @cevian's suggestion, below Hello Timescale is fine!